blackbox-tests/docker-compose.yml

```diff
@@ -14,8 +14,9 @@ services:
       - sh
       - -c
       - |
-        mongo "$$OTR_MONGO_URL" --eval 'rs.initiate({ _id: "myapp", members: [{ _id: 0, host: "mongo:27017"}] })'
         update-ca-certificates
+        openssl s_client -connect redis_tls:6380
```

Comment:
I added openssl as a dependency in blackbox-tests/Dockerfile.oplogtoredis-blackbox. This s_client call is to debug the tls connection. I'm open to downgrading this to a comment and removing the openssl dependency, since this adds a bunch of logging, but if this test breaks again it's probably the first thing I'm going to check.

Reply:
I cleaned this up - since we're only using redis 6, I removed the ssl proxy container from the docker compose test